src/components/location/dto/create-location.dto.ts

```diff
@@ -16,7 +16,10 @@ export abstract class CreateLocation {
   @Field(() => Sensitivity)
   readonly sensitivity: Sensitivity;
 
-  @Field({ nullable: true })
+  @Field({ nullable: true, description: 'Must be 3 uppercase characters' })
+  @IsAlpha()
+  @IsUppercase()
+  @Length(3, 3)
```

Comment:
Aren't they typically lowercase? It might just be better to accept either case and transform it to one or the other.

Reply:
Honestly I'm not sure.  @sethmcknight can you confirm what validation we need on this field?  When I google iso31663 the first wiki page actually says 4 letters: https://en.wikipedia.org/wiki/ISO_3166-3

Reply:
Ok I see the confusion. We are not actually referencing the ISO 3166-3 codes here, which are the deleted codes from the ISO 3166-1 database. Instead we are referencing ISO 3166-1, specifically the alpha 3 code.

Reply:
I've seen them written both ways (upper/lower). We should just normalize to one of those instead of requiring it to be one of the cases.

Reply:
Changed to take upper and lower but saved as upper case on both create and update.